mappings/net/minecraft/util/math/ChunkPos.mapping

```diff
@@ -21,6 +21,8 @@ CLASS net/minecraft/class_1923 net/minecraft/util/math/ChunkPos
 	METHOD method_19281 stream (Lnet/minecraft/class_1923;Lnet/minecraft/class_1923;)Ljava/util/stream/Stream;
 		ARG 0 pos1
 		ARG 1 pos2
+	METHOD method_24022 getChebyshevDistance (Lnet/minecraft/class_1923;)I
```

Comment:
```suggestion
	METHOD method_24022 getChebyshevDistanceTo (Lnet/minecraft/class_1923;)I
```

Reply:
`vec3i` uses `getSquaredDistance` and `getManhattanDistance`

Reply:
These were my suggestions:

- `findEnclosingBox`
- `findSmallestEnclosingBox`
- `findBoxEnclosing`
- `findSmallestBoxEnclosing`
- `findBoundingBox`

Reply:
Honestly `smallest` is pretty useless. When you say a union of two sets, you can create a union by adding extra useless stuff in it, yet we usually don't do so.

Reply:
is this about `smallestBoxContaining`?